pkg/cloudprovider/aws/fleet/launchtemplate.go

```diff
@@ -81,6 +88,11 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, clust
 	if err != nil {
 		return nil, fmt.Errorf("getting instance profile, %w", err)
 	}
+	amiID, err := p.getAMIID(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting AMI ID, %w", err)
+	}
+	zap.S().Debugf("AMI ID from SSM %s", amiID)
```

Comment:
Do you mind matching style with the other cloud provider debug statements?
`zap.S().Debugf("Successfully discovered subnets in %d zones for cluster %s", len(zonalSubnetMap), clusterName)`